test/parallel/test-fs-writestream-open-write.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+const { strictEqual } = require('assert');
+const { tmpdir } = require('os');
+const { join } = require('path');
+const fs = require('fs');
+
+// Regression test for https://github.com/nodejs/node/issues/51993
+
+const file = join(tmpdir(), `test-fs-writestream-open-write-${process.pid}.txt`);
```

Comment:
Can't/Shouldn't we use tmpdir test utils?

Reply:
@targos done.